### PR TITLE
cmd: plots: update messaging

### PR DESCRIPTION
**Pull request by pared**
* [ ] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [x] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

Related: https://github.com/iterative/dvc.org/pull/3691

### CODE REVIEW

**Review by karajan1001 — APPROVED** [review]

File: dvc/commands/plots.py [hunk 1]
```diff
@@ -270,8 +270,8 @@ def add_parser(subparsers, parent_parser):
     fix_subparsers(plots_subparsers)
 
     SHOW_HELP = (
-        "Generate plots from target files or plots definitions from "
-        "`dvc.yaml` file."
+        "Generate plots from target files or standalone plots configurations "
+        "from `dvc.yaml` file."
```
Comment (dberenbaum) [inline dvc/commands/plots.py:274]: Not sure I understand this change. There's nothing about `dvc plots show` that is specific to standalone plots, right? I think we could leave this as is.

**Comment by shcheklein** [comment]
qq - do we use this terminology in docs now - standalone and stage? 

do we need both? can we make one of them just "plots" and the second "something plots"? or even make it "plots defined within the stage"?

**Comment by dberenbaum** [comment]
> qq - do we use this terminology in docs now - standalone and stage?
> 
> do we need both? can we make one of them just "plots" and the second "something plots"? or even make it "plots defined within the stage"?

See https://github.com/iterative/dvc.org/pull/3691#discussion_r912076424 (I think there are other related comments also). Let's address in https://github.com/iterative/dvc.org/pull/3691 and hold on merging this until that is merged?

**Comment by efiop** [comment]
@pared What's the status here?

**Comment by pared** [comment]
> @pared What's the status here?

We need to keep this one in sync with docs issue. Once the discussion there is resolved we will be able to merge this.

**Comment by pared** [comment]
Addressed https://github.com/iterative/dvc.org/pull/3691#discussion_r930393174
cc @jorgeorpinel

**Comment by jorgeorpinel** [comment]
Thanks @pared !

> do we use this terminology in docs now - standalone and stage?

We use "top-level plots" or "top-level plot definitions" and "stage plots" or "plot outputs" (for now) [in docs](https://github.com/iterative/dvc.org/pull/3691/files) @shcheklein .

**Review by jorgeorpinel — APPROVED** [review]

**Pull request merged**